mlos_bench/mlos_bench/tests/config/cli/mock-bench.jsonc

```diff
@@ -14,9 +14,10 @@
 
     "environment": "environments/mock/mock_env.jsonc",
 
-    "tunable_values": [
-        "tunable-values/tunable-values-example.jsonc"
-    ],
+    // Use default values for tunables unless otherwise specified
+    // "tunable_values": [
+    //     "tunable-values/tunable-values-example.jsonc"
+    // ],
```

Comment:
Let's add a Launcher test that verifies that the implicit Optimizer and TunableGroups were initialized with the defaults.

Reply:
Or at least double check that there is one for that already.

Reply:
yep, that was the whole point. You'll see the new unit tests below.

Reply:
Ah, yep, I see it now.  Thanks!